include/libdnf/rpm/package.hpp

```diff
@@ -62,6 +62,17 @@ struct PackageId {
     int id{0};
 };
 
+struct Changelog {
+public:
+    explicit Changelog(time_t _timestamp, std::string _author, std::string _text)
```

Comment:
Why are argument names prefixed with an underscore?

Reply:
Oh, thanks. I will change this before switching from draft to regular PR.